test/MUXControlsTestApp/verification/CalendarView-4.xml

```diff
@@ -75,24 +75,24 @@
             FocusVisualPrimaryBrush=#E4000000
             Visibility=Visible
             RenderSize=284,40
-          [Windows.UI.Xaml.Controls.ContentPresenter]
-              Foreground=#E4000000
-              Padding=0,0,0,0
+          [Windows.UI.Xaml.Controls.Border]
```

Comment:
How is CalendarView the only control with updated VisualVerification Files?

Reply:
only Calendar View button Template was changed - from ContentPresenter to TextBlock